content/developer/tutorials/discover_js_framework/01_owl_components.rst

```diff
@@ -41,9 +40,11 @@ button.
 
 .. code-block:: js
 
+   /** @odoo-module **/
```

Comment:
In which version should I remove this? 17.1 or master?

Reply:
master: https://github.com/odoo/odoo/pull/142858